resources/containers/record-button.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { FaMicrophone, FaPause, FaPlay } from "react-icons/fa6";
+import record_btn_styles from "./styles/record-button.module.css";
+
+interface RecordButtonProps {
+  recordingState: "inactive" | "recording" | "paused";
+  audioLevel: number;
+  onClick: () => void;
+  statusMessage: string;
+}
+
+const RecordButton: React.FC<RecordButtonProps> = ({
+  recordingState,
+  audioLevel,
+  onClick,
+  statusMessage,
+}) => {
+  return (
+    <>
+      <button
+        className={`${record_btn_styles.recordButton} ${
+          recordingState === "recording"
+            ? record_btn_styles.recording
+            : recordingState === "paused"
+              ? record_btn_styles.paused
+              : ""
```

Comment:
To simplify this we can create a variable for it before the template and then use the variable in the template.